packages/@sanity/migrate/src/types.ts

```diff
@@ -76,9 +112,20 @@ export type DocumentMigrationReturnValue =
   | RawMutation
   | RawMutation[]
 
+/**
+ * Possible return values from a migration helper that runs on nodes within a document
+ */
 export type NodeMigrationReturnValue = DocumentMigrationReturnValue | Operation | Operation[]
 
+/**
+ * Node migration helper functions. As the migration is processing a document, it will visit each node in the document, depth-first, call the appropriate helper function for the node type and collect any mutations returned from it.
```

Comment:
```suggestion
 * @public
 *
 * Node migration helper functions. As the migration is processing a document, it will visit each node in the document, depth-first, call the appropriate helper function for the node type and collect any mutations returned from it.
```